src/main/java/marquez/MarquezContext.java

```diff
@@ -134,49 +153,17 @@ private MarquezContext(
             jobResource,
             tagResource,
             serviceExceptionMapper,
-            jdbiException);
+            jdbiException,
+            openLineageResource);
   }
 
-  public void registerListener(@NonNull RunTransitionListener listener) {
-    runTransitionListeners.add(listener);
-  }
-
-  public static Builder builder() {
-    return new Builder();
-  }
+  public ObjectMapper getLineageObjectMapper() {
+    ObjectMapper mapper = new ObjectMapper();
+    mapper.setSerializationInclusion(Include.NON_NULL);
+    mapper.registerModule(new JavaTimeModule());
+    mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+    mapper.disable(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE);
 
-  public static class Builder {
-    private Jdbi jdbi;
-    private ImmutableSet<Tag> tags;
-    private List<RunTransitionListener> runTransitionListeners;
-
-    Builder() {
-      this.tags = ImmutableSet.of();
-      this.runTransitionListeners = new ArrayList<>();
-    }
-
-    public Builder jdbi(@NonNull Jdbi jdbi) {
-      this.jdbi = jdbi;
-      return this;
-    }
-
-    public Builder tags(@NonNull ImmutableSet<Tag> tags) {
-      this.tags = tags;
-      return this;
-    }
-
-    public Builder runTransitionListener(@NonNull RunTransitionListener runTransitionListener) {
-      return runTransitionListeners(Lists.newArrayList(runTransitionListener));
-    }
-
-    public Builder runTransitionListeners(
```

Comment:
I think both `runTransitionListener()` and `runTransitionListeners()` an be helpful. Do we want to remove them?